lib/rules/one-var.js

```diff
@@ -250,6 +254,88 @@ module.exports = {
         }
 
 
+        /**
+         * Does the node have a `;` after it?
+         * @param   {ASTNode}  node The node to check
+         * @returns {boolean}       If the node has a `;` after it
+         */
+        function hasSemi(node) {
+            return sourceCode.getLastToken(node).type === "semi";
```

Comment:
I'm confused by this line because semicolon tokens have type `Punctuator`, not `semi` (e.g. see the `tokens` list in [this AST](https://astexplorer.net/#/gist/733f028f51f651b3babcc23ed8e9031e/92bf614d7e64b85fc6134c7a48057dac0c0e6c9f)). Are you sure this function ever returns `true`?

Reply:
Turns out this is dead code, so I’m going to 🔥 it.